ch.elexis.core.p2site/Elexis.product

```diff
@@ -61,16 +63,18 @@
    <configurations>
       <plugin id="ch.elexis.core.jpa.datasource" autoStart="true" startLevel="3" />
       <plugin id="ch.elexis.core.jpa.entities" autoStart="true" startLevel="3" />
-      <plugin id="ch.elexis.core.logging" autoStart="true" startLevel="2" />
+      <plugin id="ch.elexis.core.logging.osgi" autoStart="true" startLevel="2" />
```

Comment:
Bitte kurze begründung warum das hier getauscht wird!

Reply:
Das Bundle wurde im Juli 2018 mit dem commit  "[14329] moved entities to separate bundle, updated for weaving" von ch.elexis.core.logging auf ch.elexis.core.logging.osgi umbenannt, jedoch keine Anpassung dazu im Elexis.product gemacht.